contracts/MarginBase.sol

```diff
@@ -377,14 +374,10 @@ contract MarginBase is MinimalProxyable, IMarginBase, OpsReady {
 
         /// @notice impose fee
         /// @dev send fee to Kwenta's treasury
-        if (totalSizeDeltaInUSD > 0) {
+        if (tradingFee > 0) {
             require(
-                marginAsset.transfer(
-                    marginBaseSettings.treasury(),
-                    (totalSizeDeltaInUSD * marginBaseSettings.tradeFee()) /
-                        MAX_BPS
-                ),
-                "MarginBase: unable to pay fee"
+                marginAsset.transfer(marginBaseSettings.treasury(), tradingFee),
+                "MarginBase: unable to pay trading fee"
```

Comment:
Let's also emit an event here so we can track cross margin fees in a subgraph

Reply:
Okay, will add